### ISSUE TITLE
Convert latex to unicode in list command

### ISSUE THREAD

**Issue by yurivict** [pubs/pubs#177]
This article http://sci-hub.tw/10.1126/sciadv.aar8173 shows like this:
```
[Kj_r_2018] Kj\aer, Kurt H. et al. "A large impact crater beneath Hiawatha Glacier in northwest Greenland" Science Advances (2018) 
```

**Comment by omangin** [comment]
Indeed, thanks for reporting this.

**Comment by omangin** [comment]
Here I believe we can just run the latex to unicode function from bibtex-parser before pretty-printing the elements in the list. That way there is no need for additional dependencies.

**Comment by omangin** [comment]
Fixed by #185
